src/js/vendor/highlight.js

```diff
@@ -38,6 +43,21 @@ var hljs = (window.hljs = require('highlight.js/lib/core'))
   hljs.registerLanguage('swift', require('highlight.js/lib/languages/swift'))
   hljs.registerLanguage('xml', require('highlight.js/lib/languages/xml'))
   hljs.registerLanguage('yaml', require('highlight.js/lib/languages/yaml'))
+
+/* Add the additional keywords to the bash language
+ * References: https://github.com/highlightjs/highlight.js/wiki/Adding-keywords-to-a-language-at-runtime
+ * Layout used and defined in highlight.css at .hljs-built_in
+*/
+  hljs.getLanguage('bash').keywords.built_in.push(...CUSTOM_KEYWORDS);
+  // console.log(hljs.getLanguage('bash').keywords);
+
+/* To eliminate the security messages in the browser telling:
+ * "One of your code blocks includes unescaped HTML. This is a potentially serious security risk"
+ * See: https://github.com/highlightjs/highlight.js/wiki/security
+ * A hljs command has been added in partials/footer-scripts.hbs
+ * hljs.configure({ ignoreUnescapedHTML: true })
+ * This is not a security issue for us as we have a static site
```

Comment:
That is simply not true. Someone could sneak in evil code in a docs PR, and it can get merged by someone who is not aware of this potential issue. Why not simply escape the HTML within code blocks as recommended? There is no real reason to security mechanisms.

Reply:
ok, thanks for the insight. I will open a new issue.